src/api/routes/v1/code.routes.ts

```diff
@@ -1,9 +1,13 @@
 import * as express from 'express';
-import { runCode, getCode } from '../../controllers/code.controller';
+import {
+  runCode,
+  getCode,
+  submitCode
+} from '../../controllers/code.controller';
 const router = express.Router();
 
 router.route('/').post(runCode);
-
-router.route('/:runId').get(getCode);
+router.route('/submission').post(getCode);
```

Comment:
I'm sure there's a good reason, but why is this a `POST` request instead of a `GET` request?

Reply:
I need to pass in both the run id and a boolean If the output from judge0 should be base64 encoded. I didn't know if base64 encoded string followed uri parameters so I decided to make a post request instead to put in the body